docs/content/references/components.md

```diff
@@ -76,21 +76,21 @@ The container tags follow one of two patterns:
 
 For example, if pulling from the [customer portal](https://access.crunchydata.com/), the following would both be valid tags to reference the pgBouncer container:
```

Comment:
⛏️ Says "both" but there are four?

😩 I also missed this spelling of PgBouncer recently.

```suggestion
For example, if pulling from the [customer portal](https://access.crunchydata.com/), the following would all be valid tags to reference the PgBouncer container:
```

Reply:
Updated per suggestion.